motionplan/plannerOptions.go

```diff
@@ -44,8 +47,6 @@ const (
 	defaultPathStepSize = 10
 )
 
-var defaultPlanner = newCBiRRTMotionPlanner
```

Comment:
This is actually due to a compiler bug.

If I have:
```
var defaultPlanner = newCBiRRTMotionPlanner
opt.PlannerConstructor = defaultPlanner
```
then it refuses to compile due to a circular reference, as `newCBiRRTMotionPlanner` calls `newBasicPlannerOptions`.

But if I remove this line and instead simple have `opt.PlannerConstructor = newCBiRRTMotionPlanner`, then it compiles fine.

I'll add a comment to this effect.

Reply:
Cool if you could put the comment up where the rest of the defaults are defined that would be good because thats probably where someone will go to look for this information

Reply:
Done